presto-geospatial/src/test/java/com/facebook/presto/plugin/geospatial/TestGeoFunctions.java

```diff
@@ -478,6 +482,24 @@ public void testSTEnvelope()
         assertFunction("ST_AsText(ST_Envelope(ST_GeometryFromText('GEOMETRYCOLLECTION (POINT (5 1), LINESTRING (3 4, 4 4))')))", VARCHAR, "POLYGON ((3 1, 5 1, 5 4, 3 4, 3 1))");
     }
 
+    @Test
+    public void testSTEnvelopeAsPts()
+    {
+        assertEnvelopeAsPts("MULTIPOINT (1 2, 2 4, 3 6, 4 8)", new Point(1, 2), new Point(4, 8));
```

Comment:
Thanks for testing so many geometry types. For completes, let's add a test for a point: `POINT (1 2)`.

Reply:
Done